Mathlib/Geometry/Manifold/PartitionOfUnity.lean

```diff
@@ -626,64 +626,45 @@ theorem exists_contMDiffOn_section_forall_mem_convex_of_local
   have h_x_in_Umap_j : x ∈ W j := interior_subset (hρU j h_x_in_tsupport_ρj)
   exact h_mem_t j x h_x_in_Umap_j
 
-/-- Let `V` be a vector bundle over a σ-compact Hausdorff finite-dimensional topological manifold
-`M`. Let `t : M → Set (V x)` be a family of convex sets in the fibers of `V`.
-Suppose that for each point `x₀ : M` there exists a neighborhood `U_x₀` of `x₀` and a local
-section `s_loc : M → V x` such that `s_loc` is $C^∞$ smooth on `U_x₀` (when viewed as a map to
-the total space of the bundle) and `s_loc y ∈ t y` for all `y ∈ U_x₀`.
-Then there exists a global smooth section `s : Cₛ^∞⟮I_M; F_fiber, V⟯` such that
-`s x ∈ t x` for all `x : M`.
--/
-theorem exists_smooth_section_forall_mem_convex_of_local
-    {F_fiber : Type*} [NormedAddCommGroup F_fiber] [NormedSpace ℝ F_fiber]
-    (V : M → Type*) [∀ x, NormedAddCommGroup (V x)] [∀ x, Module ℝ (V x)]
-    [TopologicalSpace (TotalSpace F_fiber V)] [FiberBundle F_fiber V] [VectorBundle ℝ F_fiber V]
-    (t : ∀ x, Set (V x)) (ht_conv : ∀ x, Convex ℝ (t x))
-    (Hloc :
-      ∀ x₀ : M, ∃ U_x₀ ∈ 𝓝 x₀, ∃ (s_loc : (x : M) → V x),
-        (ContMDiffOn I (I.prod 𝓘(ℝ, F_fiber)) ∞ (fun x ↦ TotalSpace.mk' F_fiber x (s_loc x)) U_x₀) ∧
-        (∀ y ∈ U_x₀, s_loc y ∈ t y)) :
-    ∃ s : Cₛ^∞⟮I; F_fiber, V⟯, ∀ x : M, s x ∈ t x :=
-      exists_contMDiffOn_section_forall_mem_convex_of_local I V t ht_conv Hloc
+@[deprecated (since := "2025-11-06")]
+alias exists_contMDiffOn_section_forall_mem_convex_of_local :=
+  exists_contMDiffSection_forall_mem_convex_of_local
+
+@[deprecated (since := "2025-11-06")]
+alias exists_smooth_section_forall_mem_convex_of_local :=
+  exists_contMDiffSection_forall_mem_convex_of_local
 
 /-- Let `M` be a σ-compact Hausdorff finite-dimensional topological manifold. Let `t : M → Set F`
 be a family of convex sets. Suppose that for each point `x : M` there exists a neighborhood
 `U ∈ 𝓝 x` and a function `g : M → F` such that `g` is $C^n$ smooth on `U` and `g y ∈ t y` for all
 `y ∈ U`. Then there exists a $C^n$ smooth function `g : C^n⟮I, M; 𝓘(ℝ, F), F⟯` such that `g x ∈ t x`
 for all `x`.
 
-This is a special case of `exists_contMDiffOn_section_forall_mem_convex_of_local` where `V` is the
-trivial bundle. See also `exists_smooth_forall_mem_convex_of_local` and
-`exists_smooth_forall_mem_convex_of_local_const`. -/
-theorem exists_contMDiffOn_forall_mem_convex_of_local (ht : ∀ x, Convex ℝ (t x))
+This is a special case of `exists_contMDiffSection_forall_mem_convex_of_local` where `V` is the
+trivial bundle. See also `exists_smooth_forall_mem_convex_of_local_const`. -/
```

Comment:
Can you update this line also?

Reply:
I think `exists_smooth_forall_mem_convex_of_local_const` is the correct one: since you can get a C^\infty function in this case, I did not change the statement.

Reply:
I think that `smooth` for C^\infty is standard in mathlib -- for instance, we have SmoothBumpFunction.

Reply:
We do --- but we renamed `SmoothSection` to `ContMDiffSection`. (That one has a regularity parameter, though.)